package/kedro_viz/models/flowchart.py

```diff
@@ -555,13 +555,18 @@ class DataNodeMetadata(GraphNodeMetadata):
     # command to run the pipeline to this data node
     run_command: Optional[str] = field(init=False, default=None)
 
+    preview: Optional[Dict] = field(init=False, default=None)
+
     # TODO: improve this scheme.
     def __post_init__(self, data_node: DataNode):
         self.type = data_node.dataset_type
         dataset = cast(AbstractDataSet, data_node.kedro_obj)
         dataset_description = dataset._describe()
         self.filepath = _parse_filepath(dataset_description)
 
+        if (self.type == "pandas.csv_dataset.CSVDataSet" or self.type == "pandas.parquet_dataset.ParquetDataSet" or self.type == "pandas.excel_dataset.ExcelDataSet"):
```

Comment:
I was thinking maybe we could do this 

PREVIEW_DATASETS = ['"pandas.csv_dataset.CSVDataSet",
    "pandas.parquet_dataset.ParquetDataSet",
    "pandas.excel_dataset.ExcelDataSet"] on top of the file

and then let's do isinstance(self.type, PREVIEW_DATASET)

Reply:
ah yeah good point, i've updated this but use ` if self.type in PREVIEW_DATASETS:` instead of `isInstance`